docs/alps/hardware.md

```diff
@@ -40,20 +40,36 @@ Alps was installed in phases, starting with the installation of 1024 AMD Rome du
 
 There are currently four node types in Alps, with another becoming available in 2025:
 
-| type           | blades | nodes | CPU sockets | GPU devices |
-| ----           | ------:| -----:| -----------:| -----------:|
-| NVIDIA GH200   | 1344   | 2688  | 10,752      | 10,752      |
-| AMD Rome       |  256   | 1024  |  2,048      | --          |
-| NVIDIA A100    |   72   |  144  |    144      | 576         |
-| AMD MI250x     |   12   |   24  |     24      |  96         |
-| AMD MI300A     |   64   |  128  |    512      | 512         |
+| type           | abbreviation  | blades | nodes | CPU sockets | GPU devices |
+| ----           | -------       | ------:| -----:| -----------:| -----------:|
+| NVIDIA GH200   | gh200         | 1344   | 2688  | 10,752      | 10,752      |
+| AMD Rome       | zen2          |  256   | 1024  |  2,048      | --          |
+| NVIDIA A100    | a100          |   72   |  144  |    144      | 576         |
+| AMD MI250x     | mi200         |   12   |   24  |     24      |  96         |
+| AMD MI300A     | mi300         |   64   |  128  |    512      | 512         |
 
 [](){#ref-alps-gh200-node}
 ### NVIDIA GH200 GPU Nodes
 
-!!! todo
+There are 24 cabinets, in 4 rows with 6 cabinets per row:
+
+* 8 chassis per cabinet
+* 7 blades per chassis
+    * a chassis can contain up to 8 blades, however Alps' gh200 chassis are underpopulated so that we can increase the amount of power delivered to each node.
```

Comment:
This is interesting information, but it might be better in a note after the list describing the system as it is?